premailer/premailer.py

```diff
@@ -402,25 +402,23 @@ def transform(self, pretty_print=True, **kwargs):
         # URLs
         #
         if self.base_url:
-            if not self.base_url.endswith('/'):
-                self.base_url += '/'
+            if not urlparse(self.base_url).scheme:
+                raise ValueError('Base URL must start have a protocol')
```

Comment:
Ha! My bad. I thought he pointed out that I had put "schema" (not "scheme") in the commit message. But no, I had put "schema" in the commit message and "protocol" there in the ValueError message. Fixed now.